quantumdb-core/src/main/java/io/quantumdb/core/schema/definitions/Table.java

```diff
@@ -3,10 +3,7 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
 
-import java.util.Collection;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Don't use star imports. They tend to (auto-)import the wrong things sometimes. Instead be specific in what you want to import. If you're using IntelliJ, you can enable the "Use single class import" option in the preferences: Editor -> Code style -> Java -> Imports

Reply:
I went to that setting and it was already selected. I then noticed that the option "Class count to use import with \"*\"" was set to 5, I changed it to something higher such as 20.

Do you have a code style settings file (Eclipse or IntelliJ) that I can import? There are various other things that change whenever I "reformat" code.

Reply:
Created a gist for [my current one](https://gist.github.com/michaeldejong/f5c4b55f6c345871b96c068c976551ac). But IntelliJ does seem to have updated the defaults over time, so it probably won't be an exact match to quantumdb's. Probably best to only format LoC that change in a given commit.